adapters/restclient/lib/opentelemetry/adapters/restclient/patches/request.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module RestClient
+      module Patches
+        # Module to prepend to RestClient::Request for instrumentation
+        module Request
+          def execute(&block)
+            trace_request do |_span|
+              super(&block)
+            end
+          end
+
+          private
+
+          def trace_request # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+            http_method = method.upcase
+            span = tracer.start_span(
+              "HTTP #{http_method}",
```

Comment:
I'm not sure that I am crazy about this span name either. Ultimately we are going to want the http quantization that is currently in the Rack PR to help us name things in accordance with OpenTelemetry guidelines. We should discuss what we should do in the interim.

Reply:
I think using just `{verb}` or `{verb} {host}` would be good interim solutions. The latter might be best if it's not too much work, since the goal is to replace it soon anyways.